nucypher/blockchain/eth/deployers.py

```diff
@@ -211,6 +211,12 @@ def retarget(self, new_target: str, existing_secret_plaintext: bytes, new_secret
         _receipt = self.blockchain.wait_for_receipt(txhash=txhash)
         return txhash
 
+    def rollback(self, existing_secret_plaintext: bytes, new_secret_hash: bytes) -> bytes:
+        origin_args = {'from': self.deployer_address}  # FIXME
```

Comment:
What's this `FIXME` in reference to?

Reply:
Gas management.

Reply:
<https://github.com/nucypher/nucypher/issues/842>